sunpy/coordinates/tests/test_transformations.py

```diff
@@ -96,7 +96,7 @@ def test_hcrs_hgs():
     earth_hgs = earth_hcrs.transform_to(HeliographicStonyhurst)
 
     # The HGS longitude of the Earth should be zero within numerical error
-    assert quantity_allclose(earth_hgs.lon, 0*u.deg, atol=1e-12*u.deg)
+    assert quantity_allclose(earth_hgs.lon % (360*u.deg), 0*u.deg, atol=1e-12*u.deg)
```

Comment:
To be nitpicky here: this only fixes the case that `earth_hgs.lon` is slightly larger that 360°. But it may also happen that it is (360-1e-12) degrees, which would make the failure re-appear.

Reply:
Yeah, I think that is probably what I will do. 

Here (https://github.com/sunpy/sunpy/pull/2930/files) we tried wrapping shifting it to 180 but it fails on my local machine.